src/elements.js

```diff
@@ -123,7 +126,7 @@ function resolveObj(resolver, defs) {
   for (const prop of Object.keys(defs)) {
     const optDefs = defs[prop];
     const value = resolver[prop];
-    result[prop] = isObject(optDefs) ? resolveObj(value, optDefs) : value;
+    result[prop] = inspect(prop, value, optDefs) ? resolveObj(value, optDefs) : value;
```

Comment:
Should work without the inpect, if its described as `indexable: false`

Reply:
All options are defined as indexable: false and in fact it's working with `color` options when defined as array. But the `font` is a special case, because is an array of objects which need to be resolved as well.

Reply:
I have removed the `inspect` function but I need to check if the property is `indexable` (function anyway needed to set the `_indexable` property in the annotations descriptors.